src/DynamoCore/Logging/DynamoAnalyticsClient.cs

```diff
@@ -169,7 +175,7 @@ public DynamoAnalyticsClient(DynamoModel dynamoModel)
             //Dynamo app version.
             var appversion = dynamoModel.AppVersion;
 
-            product = new ProductInfo() { Name = "Dynamo", VersionString = appversion, AppVersion = dynamoModel.Version };
+            product = new ProductInfo() { Id = "DYN", Name = "Dynamo", VersionString = appversion, AppVersion = dynamoModel.Version, BuildId = "CB159AA3-2F20-4791-AE8C-8939E90C9600", ReleaseId = "2.5.0", MasterId = "{4C7B248C-56D3-488F-8E23-8CAFD2651968}" };
```

Comment:
is this going to effect the gathered google analytics data?

Reply:
The added data should not interfere with Google analytics.

It is still designed to simplify internal testing (just test dummy identifiers). 
We need to decide on proper UPI (universal product identifier) configs ...then we can properly supply them here